pages/caps/novosusuarios/index.js

```diff
@@ -5,6 +5,7 @@ import { useEffect, useMemo, useState } from 'react';
 import Select from 'react-select';
 import { v1 as uuidv1 } from 'uuid';
 import { redirectHomeNotLooged } from '../../../helpers/RedirectHome';
+import { TabelaCid, TabelaDetalhamentoPorCaps } from "../../../components/Tabelas";
```

Comment:
A tabela de Detalhamento por caps só é usada na página de Perfil do usuário. Pode remover a importação dela aqui, por favor?

```suggestion
import { TabelaCid } from "../../../components/Tabelas";
```